src/content/guides/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Ethereum Guides
+description: A collection of practical guides explaining the basics of using crypto for beginners.
+lang: en
+---
+
+# Ethereum Guides
+
+Do you want to start your crypto journey? It can be difficult to understand and use all the new technology. These guides will teach you how to get started step by step in a practical manner.
```

Comment:
```suggestion
Do you want to start your Ethereum journey? It can be confusing to use and understand new technology. Our guides will teach you how to get started step-by-step.
```